nixos/modules/system/boot/plymouth.nix

```diff
@@ -65,6 +66,15 @@ in
         '';
       };
 
+      extraConfig = mkOption {
+        type = types.lines;
+        default = "";
+        description = ''
+          Literal string to append to <literal>configFile</literal>
```

Comment:
`<literal>configFile</literal>` ->  `<literal>/etc/plymouth/plymouthd.conf</literal>`

`configFile` is not an exposed public option, it should be mentioned in the module's public interface documentation.